addons/mass_mailing_crm/data/mass_mailing_demo.xml

```diff
@@ -14,6 +14,7 @@
         <field name="reply_to_mode">new</field>
         <field name="reply_to">{{ object.company_id.email }}</field>
         <field name="body_html" type="html">
+<div class="oe_structure o_editable">
```

Comment:
Isn't it breaking translations ?

Reply:
@tde-banana-odoo I have limited experience with translations, but I have checked the `.pot` files and none of the views themselves have translations. Instead, phrases from the mail body/content are translated. Since I modify the view structure and not the mail body, I think there shouldn't be any problem. (+ this is demo data)

Reply:
Sorry, actually since a few versions mailing are not translatable anymore :D (translate=false) so my remark does not apply.